### PR TITLE
=str 19921 Error in documentation: Graph_cycles

### PR DESCRIPTION
**Pull request by agolubev**
added one paragraph to doc to clarify that examples are abstract and graphs actually do no run because of problems in each snippet.
Ref: 19921

### CODE REVIEW

File: akka-docs/rst/scala/stream/stream-graphs.rst [hunk 1]
```diff
@@ -292,6 +292,9 @@ Cycles in bounded stream topologies need special considerations to avoid potenti
 This section shows several examples of problems that can arise from the presence of feedback arcs in stream processing
 graphs.
 
+In examples below runnable graphs are created but do not run because each have some issue and will not complete normally.
```
Comment (ktoso) [inline akka-docs/rst/scala/stream/stream-graphs.rst]: In the following examples...

Comment (ktoso) [inline akka-docs/rst/scala/stream/stream-graphs.rst]: Not sure about wording on "complete normally", perhaps call out that they deadlock?

**Comment by ktoso** [comment]
OK TO TES

**Comment by ktoso** [comment]
OK TO TEST

**Comment by ktoso** [comment]
LGTM, good wording now, thanks! 👍

**Comment by ktoso** [comment]
Refs #19921

File: akka-docs/rst/java/stream/stream-graphs.rst [hunk 2]
```diff
@@ -235,6 +235,9 @@ Cycles in bounded stream topologies need special considerations to avoid potenti
 This section shows several examples of problems that can arise from the presence of feedback arcs in stream processing
 graphs.
 
+In the following examples runnable graphs are created but do not run because each have some issue and will deadlock after start.
+``Source`` variable is not defined as the nature and number of element does not matter for described problems.
```
Comment (drewhk) [inline akka-docs/rst/java/stream/stream-graphs.rst]: "The Source"
"nature and number of elements"
"for the described problems"

Comment (agolubev) [inline akka-docs/rst/java/stream/stream-graphs.rst]: @drewhk  It's been merged. Should I create another ticket?

Comment (drewhk) [inline akka-docs/rst/java/stream/stream-graphs.rst]: If you have another PR I think it does not hurt to fix there, falls under "boy-scout" IMHO and does not deserve a dedicated ticket (we have tons of tickets already :) ).

**Pull request merged**